src/generateDapp.ts

```diff
@@ -138,6 +138,63 @@ export async function generateDapp(selection: Selections) {
       case "boilerplate-template":
         await generateEnvFile();
         break;
+      case "clicker-game-tg-mini-app-template":
+        const dir = "frontend/components";
+
+        if (selection.signingOption === "explicit") {
+          copy(
```

Comment:
any reason not to use https://github.com/aptos-labs/create-aptos-dapp/blob/main/src/utils/helpers.ts#L20 ?

Reply:
i am using that `copy`